src/test/java/ca/uhn/fhir/jpa/starter/ExampleServerR5IT.java

```diff
@@ -132,5 +137,10 @@ void beforeEach() {
     String ourServerBase = "http://localhost:" + port + "/fhir/";
     ourClient = ourCtx.newRestfulGenericClient(ourServerBase);
     ourClient.registerInterceptor(new LoggingInterceptor(true));
+ // Create an HTTP basic auth interceptor
+ 	String username = "hapi";
+ 	String password = "hapi123";
+ 	IClientInterceptor authInterceptor = new BasicAuthInterceptor(username, password);
+ 	ourClient.registerInterceptor(authInterceptor);
```

Comment:
@gitcliff , revert these changes ..these are just example servers ..not the actual running servers